### PR TITLE
Javascript-Operators-and-Expressions.

### PR DESCRIPTION
**Pull request by Mitchelllund**
Screenshot 2022-05-31 202624js-operators-and

### CODE REVIEW

File: javascript-operators-and-expressions/main.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+console.log('typeof isAcidic:', typeof isAcidic);
+
+var headCount = 243;
+var isSparta = headCount === 300;
+
+console.log('isSparta:', isSparta);
+console.log('typeof isSparta:', typeof isSparta);
+
+var motto = fullName;
+motto = motto + ' is the GOAT';
```
Comment (BrettAlbright) [inline javascript-operators-and-expressions/main.js]: you should be using the `+=` operator here.

**Review by BrettAlbright — CHANGES_REQUESTED** [review]
small update needed

**Comment by Mitchelllund** [comment]
Screenshot 2022-05-31 202624js-operators-and

**Review by BrettAlbright — APPROVED** [review]
👍🏼

**Pull request merged**